central/complianceoperator/v2/benchmarks/datastore/datastore_impl_test.go

```diff
@@ -57,16 +57,62 @@ func (s *complianceBenchmarkDataStoreSuite) SetupTest() {
 	s.noAccessCtx = sac.WithGlobalAccessScopeChecker(context.Background(), sac.DenyAllAccessScopeChecker())
 
 	s.mockCtrl = gomock.NewController(s.T())
+	s.T().Setenv("POSTGRES_PORT", "5432")
+	s.T().Setenv("POSTGRES_PASSWORD", "password")
+	s.T().Setenv("USER", "postgres")
 
-	s.db = pgtest.ForT(s.T())
+	s.db = pgtest.ForTCustomDB(s.T(), "central")
 	s.storage = benchmarkStorage.New(s.db)
-	s.datastore = New(s.storage)
+	s.datastore = &datastoreImpl{
+		store: s.storage,
+		db:    s.db,
+	}
+
 }
 
 func (s *complianceBenchmarkDataStoreSuite) TearDownTest() {
 	s.db.Teardown(s.T())
 }
 
+func (s *complianceBenchmarkDataStoreSuite) TestGetControl() {
+	// TODO: Should be moved to rule datastore? Test does not work, nil pointer in Database and missing permission in SAC?
```

Comment:
This is interesting.  Probably because there is no cluster.  Being purely a lookup table, maybe benchmarks should not be associated with the compliance resource.